client/src/App.js

```diff
@@ -1,11 +1,56 @@
+import React, { useState } from "react";
+import AddNewVideo from "./AddNewVideo";
 import "./App.css";
+import dataVideo from "./exampleresponse.json";
+import RenderVideo from "./RenderVideo";
 
 function App() {
+  const [allVideos, setVideoData] = useState(dataVideo);
+  const [addVideo, setAddVideo] = useState(false);
+
+  //Removing 1 video
+  // const deleteViedeos = (deletedViedeoId) => {
+  //   setVideoData(
+  //     allVideos.filter((video) => {
+  //       return video.id !== deletedViedeoId;
+  //     })
+  //   );
+  // }
```

Comment:
Again, if you are not going to use this commented out code, it is best to just delete it in order to simplify your codebase.